include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -145,12 +146,13 @@ class OpKernelContext {
   */
   Fence_t OutputFence(int index) const;
 
+  const MLValue* GetInputMLValue(int index) const;
+  MLValue* GetOutputMLValue(int index);
+
```

Comment:
Why does a custom op need access to the raw MLValue?

Reply:
Because we expose OrtValue* types to the C API interfaces and those are MLValue* types internally. Did you know a better way?